cmd/install/install.go

```diff
@@ -183,6 +183,7 @@ func NewCommand() *cobra.Command {
 	opts.ExternalDNSImage = ExternalDNSImage
 	opts.CertRotationScale = 24 * time.Hour
 	opts.EnableSizeTagging = false
+	opts.ManagedService = hyperv1.AroHCP
```

Comment:
wouldn't this enable shared-ingress for all platforms?

Reply:
indeed. Fixed.